frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelConfig.java

```diff
@@ -44,6 +44,16 @@ public class ModelConfig {
      * timeout. default: 0 which means no timeout (ie. clientExpireTS default value Long.MAX_VALUE.
      */
     private long clientTimeoutInMills;
+    /**
+     * the job queue size of a model. By default, job_queue_size is set as 100 in config.property
+     * for all models. Here, jobQueueSize: -1 means no customized setting for the model.
+     */
+    private int jobQueueSize;
```

Comment:
For my information, where do we default it to 100?

Reply:
the default queueSize  100 is passed into [Model](https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java#L66). This customized config value is used to overwrite the queueSize in this PR.